compact.go

```diff
@@ -774,12 +784,21 @@ func (c *LeveledCompactor) populateBlock(blocks []BlockReader, meta *BlockMeta,
 				}
 
 				it := &deletedIterator{it: chk.Chunk.Iterator(), intervals: dranges}
+
+				var (
```

Comment:
Why are you doing this? The Go compiler is smart enough to handle this itself

Reply:
because we need it to set the chunk meta maxt later
`chks[i].MaxTime = t`